packages/react-dom/src/server/ReactPartialRenderer.js

```diff
@@ -1327,12 +1327,12 @@ class ReactDOMServerRenderer {
     const tag = element.type.toLowerCase();
 
     let namespace = parentNamespace;
-    if (parentNamespace === Namespaces.html) {
+    if (parentNamespace === HTML_NAMESPACE) {
       namespace = getIntrinsicNamespace(tag);
     }
 
     if (__DEV__) {
-      if (namespace === Namespaces.html) {
+      if (namespace === HTML_NAMESPACE) {
```

Comment:
So this is really weird. We track the namespace in prod and the only thing we use it for is this warning.

We don't need the namespace on the server since it's implied by the HTML.

Reply:
Looks like it was intentional. https://github.com/facebook/react/pull/10452

But I'm just going to remove this because it's not worth the overhead/complexity.